main/include/SettingsBluetooth.h

```diff
@@ -0,0 +1,16 @@
+#ifndef __SETTINGS_BLUETOOTH_H__
+#define __SETTINGS_BLUETOOTH_H__
+
+// JoyCon(L) Pink
+// #define C110P_CONTROLLER_LEFT_MAC_ADDRESS             "5C:52:1E:FF:6E:A2"
+
+// JoyCon(R) Green
+// #define C110P_CONTROLLER_RIGHT_MAC_ADDRESS            "5C:52:1E:FF:51:45"
+
+// JoyCon(L) White
+#define C110P_CONTROLLER_LEFT_MAC_ADDRESS           "98:E6:B9:62:6E:58"
+
+// JoyCon(R) Gray
+#define C110P_CONTROLLER_RIGHT_MAC_ADDRESS          "98:E6:B9:5A:CA:74"
```

Comment:
We think these lines should be private.
```suggestion
// #define C110P_CONTROLLER_LEFT_MAC_ADDRESS             "0A:00:0A:AA:0A:A0"

// JoyCon(R) Green
// #define C110P_CONTROLLER_RIGHT_MAC_ADDRESS            "0A:00:0A:AA:0A:A1"

// JoyCon(L) White
#define C110P_CONTROLLER_LEFT_MAC_ADDRESS           "0A:00:0A:AA:0A:A2"

// JoyCon(R) Gray
#define C110P_CONTROLLER_RIGHT_MAC_ADDRESS          "0A:00:0A:AA:0A:A3"
```